lib/exchanger/operations/get_mail_tips.rb

```diff
@@ -0,0 +1,61 @@
+module Exchanger
+  # Getting Users Mail Tips
+  # 
+  # http://msdn.microsoft.com/en-us/library/dd899452(v=exchg.140).aspx
+  class GetMailTips < Operation
+
+    class Request < Operation::Request
+
+      attr_accessor :sender, :recipients
+
+      # Reset request options to defaults.
+      def reset
+        @sender     = 'test.test@test.com'
+        @recipients = [ 'test.test@test.com', 'test.test@test.com', 'test.test@test.com' ]
+      end
+
+      def to_xml
+        Nokogiri::XML::Builder.new do |xml|
+          xml.send("soap:Envelope", "xmlns:xsi" => NS["xsi"], "xmlns:xsd" => NS["xsd"], "xmlns:soap" => NS["soap"], "xmlns:t" => NS["t"]) do
+            xml.send("soap:Header") do
+            	xml.send("t:RequestServerVersion", "Version" => "Exchange2010")
+            end
+            xml.send("soap:Body") do
+              xml.send("GetMailTips", "xmlns" => NS["m"]) do
+                xml.send("SendingAs") do
+                  xml.send("t:EmailAddress", @sender)
+                  xml.send("t:RoutingType", "SMTP")
+                end
+                xml.send("Recipients") do
+                	@recipients.each do |recipient|
+                		xml.send("t:Mailbox") do
+                 			xml.send("t:EmailAddress", recipient)
+                  		xml.send("t:RoutingType", "SMTP")                	
+                		end
+                	end
+                end
+                xml.send("MailTipsRequested", "OutOfOfficeMessage")
+              end
+            end
+          end
+        end	
+      end
+
+    end
+
+
+    class Response < Operation::Response
+
+      def out_of_office
+        to_xml.xpath("//m:MailTips", "xmlns:m" => NS["m"]).map do |node|
+          email_address = node.xpath("t:RecipientAddress", "xmlns:t" => NS["t"])[0].children[1].text
+          out_of_office = node.xpath("t:OutOfOffice", "xmlns:t" => NS["t"]).children.children.to_s.eql?("<t:Message/>") ? false : true
+          
+          {email: email_address, out_of_office: out_of_office}
```

Comment:
I think it would be cleaner to create `MailTips` element class, something like:

```
module Exchanger
  class MailTips < Element
    element :recipient_address
    element :out_of_office, :type => Boolean
  end
end
```